frontend/src/MakerApp.tsx

```diff
@@ -37,9 +39,14 @@ export default function App() {
     const priceInfo = useLatestEvent<PriceInfo>(source, "quote");
 
     const toast = useToast();
-    let [minQuantity, setMinQuantity] = useState<string>("100");
-    let [maxQuantity, setMaxQuantity] = useState<string>("1000");
-    let [orderPrice, setOrderPrice] = useState<string>("10000");
+    let [minQuantity, setMinQuantity] = useState<string>("10");
+    let [maxQuantity, setMaxQuantity] = useState<string>("100");
+    let [orderPrice, setOrderPrice] = useState<string>("0");
+
+    const spread = 1.03;
+    if (priceInfo && orderPrice === "0") {
```

Comment:
This is not ideal, you shouldn't call these hook functions as part of the regular render code because they cause side-effects and rendering should be pure.

I think we should solve this by tracking the _real state_ that we care about which seems to be whether or not the user has entered a price themselves yet. Based on that, we can derive the price to display from the `priceInfo` or from whatever the user entered.